.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: node_js
+node_js:
+- '6'
+- '7'
+- '8'
```

Comment:
We'll have to add a polyfill then, if not add a note that this works only on node 8.